playbook/app/pb_kits/playbook/pb_icon/_icon.tsx

```diff
@@ -135,11 +134,11 @@ const Icon = (props: IconProps) => {
               className="pbiconhere"
               id={id}
           >
-            <IconFunction />
+            { React.cloneElement(iconElement) }
```

Comment:
Clone the element (allows for sending custom props like colors later) and render it as React element.